Resources/config/menu.xml

```diff
@@ -13,6 +13,7 @@
         <service id="sonata.admin.menu.group_provider" class="Sonata\AdminBundle\Menu\Provider\GroupMenuProvider" public="false">
             <argument type="service" id="knp_menu.factory"/>
             <argument type="service" id="sonata.admin.pool"/>
+            <argument type="service" id="security.authorization_checker"/>
```

Comment:
I think there might be a need for an empty argument `<argument />`, to be replaced.